stdlib/3/typing.pyi

```diff
@@ -169,8 +169,13 @@ class Generator(Iterator[_T_co], Generic[_T_co, _T_contra, _V_co]):
     @abstractmethod
     def send(self, value: _T_contra) -> _T_co: ...
 
+    @overload
     @abstractmethod
-    def throw(self, typ: Type[BaseException], val: Optional[BaseException] = ...,
+    def throw(self, typ: Type[BaseException], val: Union[BaseException, object] = ...,
+              tb: Optional[TracebackType] = ...) -> _T_co: ...
```

Comment:
Also, do you prefer separate overloads here instead of the `Union`? The different types of arguments supported for `val` are `None`, `BaseException` and finally `object`. I'm wondering if it's more readable that way.

Reply:
I think the union is fine. Overloads are very verbose, so things are generally more readable without them.

Reply:
Cool :)